static/app/views/organizationContext.tsx

```diff
@@ -202,7 +202,7 @@ class OrganizationContextContainer extends React.Component<Props, State> {
       this.props.api,
       OrganizationContextContainer.getOrganizationSlug(this.props),
       true,
-      true
+      false
```

Comment:
Can you validate that the new user onboarding experience still works OK?

Reply:
just tested it! seems to be working as expected